glossary.yml

```diff
@@ -4356,14 +4423,14 @@
     term: "relative error"
     def: >
       The absolute value of the difference between the actual and correct value
-      divided by the desired value. For example, if the actual value is 9 and the
+      divided by the correct value. For example, if the actual value is 9 and the
       correct value is 10, the relative error is 0.1. Relative error is usually more
       useful than [absolute error](#absolute_error).
   pt:
     term: "erro relativo"
     def: >
       O valor absoluto da diferença entre um valor observado e o valor correto,
-      dividido pelo vaor desejado. Por exemplo, se o valor observado é 9 e o correto é
+      dividido pelo vaor correto. Por exemplo, se o valor observado é 9 e o correto é
```

Comment:
@beatrizmilz could you please check this change?

Reply:
@beatrizmilz For context: the change was made to reflect one made in the English definition. The word 'desired' was used, but in a statistical context, there are actual values, and there are correct, or expected, values. Desire has no place here.